packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -71,326 +36,415 @@
+                                            >
+                                                <span
+                                                    class="ovh-font ovh-font-check"
+                                                    aria-hidden="true"
+                                                ></span>
+                                            </button>
+                                            <button
+                                                class="btn btn-default ml-2"
+                                                type="button"
+                                                title="{{$ctrl.deviceIdToLink}} {{::'overTheBox_common_table_cancel_modifications' | translate}}"
+                                                data-ng-disabled="!$ctrl.device.deviceId"
+                                                data-ng-click="$ctrl.device.editMode = false"
+                                            >
+                                                <span
+                                                    class="ovh-font ovh-font-wrong"
+                                                    aria-hidden="true"
+                                                ></span>
+                                            </button>
+                                        </div>
+                                    </form>
+                                </div>
                             </div>
                         </div>
-                    </div>
-                    <div class="oui-tile__item">
-                        <div class="oui-tile__definition">
-                            <div
-                                class="oui-tile__term"
-                                data-ng-if="$ctrl.serviceInfos.renew.deleteAtExpiration"
-                                data-translate="overTheBox_expiration_date"
-                            ></div>
+                        <div class="oui-tile__item">
                             <div
-                                class="oui-tile__term"
-                                data-ng-if="!$ctrl.serviceInfos.renew.deleteAtExpiration"
-                                data-translate="overTheBox_billing_date"
-                            ></div>
-                            <div class="oui-tile__description">
+                                class="oui-tile__definition"
+                                data-ng-if="$ctrl.loaders.device || !$ctrl.error.noDeviceLinked"
+                            >
+                                <span class="oui-tile__term d-block">
+                                    <span data-translate="overTheBox_publicIp">
+                                    </span>
+                                    <span
+                                        class="oui-icon oui-icon-lock"
+                                        data-ng-if="$ctrl.serviceIP.status === 'locked'"
```

Comment:
```suggestion
                                        data-ng-if="$ctrl.serviceIP.status === $ctrl.MY_STATUS_CONSTANT.locked"
```
Move the constant in constant file